bees/execbee/execbeefactory.go

```diff
@@ -103,6 +104,11 @@ func (factory *ExecBeeFactory) Actions() []bees.ActionDescriptor {
 					Description: "command to be executed",
 					Type:        "string",
 					Mandatory:   true,
+				}, {
```

Comment:
Please make sure to always run gofmt on your sources before committing them. This will avoid formatting errors like this one.

Reply:
Sorry, somehow vscode used goreturns after an update

Reply:
goreturns should actually be fine, as I believe it also uses gofmt internally. Are you sure it caused that styling?

Reply:
It seems there's more to it. I'll try to fix that later

Reply:
It seems that gofmt accepts all those syntaxes:

`}, {`

```
},
{
```

```
},

{
```

at least on my machine, it only changed the indents, not the newlines. Only if there is more linebreaks than in the last case, those get removed.

Reply:
Wow, you are right. Funny that I've noticed that ambiguous behavior before.